docs/user_guide/hyperopt.md

```diff
@@ -0,0 +1,6 @@
+Ludwig supports hyperparameter optimization using [Ray Tune](https://docs.ray.io/en/latest/tune/index.html) or a local executor.
+
+The hyperparameter optimization strategy is specified as part of the Ludwig configuration and run using
+the `ludwig hyperopt` command. Every parameter within the config can be tuned using hyperopt.
+
+The full configuration specification can be found [here](/ludwig-docs/configuration/hyperparameter_optimization).
```

Comment:
It seems like all of the absolute path links are broken for me when I try running this locally.

This link, for example, directs to an invalid 404:

> http://localhost:8000/ludwig-docs/configuration/hyperparameter_optimization

The right link seems to be:

> http://localhost:8000/0.4/configuration/hyperparameter_optimization

It seems like we may need to keep the relative paths, i.e. "../configuration/hyperparameter_optimization".

Reply:
Hmm, yeah, seems using relative paths is recommended by mkdocs. Good catch!